core/src/main/java/bio/terra/pearl/core/service/address/SmartyClient.java

```diff
@@ -16,18 +16,47 @@ public class SmartyClient {
     private com.smartystreets.api.international_street.Client internationalClient;
 
     SmartyClient(AddressValidationConfig config) {
-        ClientBuilder builder = new ClientBuilder(
+        ClientBuilder usBuilder = new ClientBuilder(
                 new StaticCredentials(config.getAuthId(), config.getAuthToken())
         );
-        usClient = builder.buildUsStreetApiClient();
-        internationalClient = builder.buildInternationalStreetApiClient();
+        ClientBuilder internationalBuilder = new ClientBuilder(
+                new StaticCredentials(config.getAuthId(), config.getAuthToken())
+        );
+        usBuilder.withLicenses(List.of("us-core-cloud"));
+        internationalBuilder.withLicenses(List.of("international-geocoding-cloud"));
+        usClient = usBuilder.buildUsStreetApiClient();
+        internationalClient = internationalBuilder.buildInternationalStreetApiClient();
     }
 
-    void send(com.smartystreets.api.us_street.Lookup lookup) throws SmartyException, IOException, InterruptedException {
-        this.usClient.send(lookup);
+    void send(com.smartystreets.api.us_street.Lookup lookup) throws AddressValidationException {
+        try {
+            this.usClient.send(lookup);
+        } catch (Exception e) {
+            handleException(e);
+            lookup.setResult(null);
+        }
     }
 
-    void send(com.smartystreets.api.international_street.Lookup lookup) throws SmartyException, IOException, InterruptedException {
-        this.internationalClient.send(lookup);
+    void send(com.smartystreets.api.international_street.Lookup lookup) throws AddressValidationException {
+        try {
+            this.internationalClient.send(lookup);
+        } catch (Exception e) {
+            handleException(e);
+            lookup.setResult(null);
+        }
     }
+
+    void handleException(Exception e) throws AddressValidationException {
+        for (String key : INVALID_ADDRESS_EXCEPTIONS) {
+            if (e.getMessage().contains(key)) {
```

Comment:
My thought is that most invalid states should be in the validation result and only errors which signal that we are interfacing with smarty wrong should be exceptions. These (`"GET request lacked required fields"` and `"GET request lacked a street field"`) happen in situations based on invalid user input. Without this code, if the user misspells a country (which, to be fair, shouldn't happen very often with the dropdown) it would throw a bad request and just have an error toast which I found confusing.